src/tabs/failsafe.html

```diff
@@ -47,7 +47,7 @@
                     <div class="spacer_box">
                         <div class="number">
                             <label>
-                                <select class="switchMode" name="failsafe_switch_mode">
+                                <select id="failsafeSwitchSelect" class="switchMode" name="failsafe_switch_mode">
```

Comment:
We don't need special ids for this. In general `name` is a good unique property and is required for accesibility etc.
```suggestion
                                <select class="switchMode" name="failsafe_switch_mode">
```